### PR TITLE
Merge beta-branch with beta-stage

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 <div class="custom-tabs heightVH-100">
     <tabset>
-        <tab heading="Basic title" id="tab1">
+        <tab heading="Generated Bills" id="tab1">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.html:16]: Please update this in language JSON.

File: apps/web-giddh/src/app/services/authentication.service.ts [hunk 2]
```diff
@@ -369,4 +369,25 @@ export class AuthenticationService {
...
+        let args: any = { headers: {} };
+        args.headers['cache-control'] = 'no-cache';
+        args.headers['Content-Type'] = 'application/xml';
+        // args.headers['Accept'] = 'application/xml';
+        args.headers = new HttpHeaders(args.headers);
+        return this._httpClient.get('https://s3-ap-south-1.amazonaws.com/giddh-app-builds/latest-mac.yml', {
+            headers: args.headers,
+            responseType: 'text'
+        }).pipe(map((res) => {
+            // let data: BaseResponse<VerifyEmailResponseModel, LinkedInRequestModel> = res as BaseResponse<any, any>;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/services/authentication.service.ts:389]: remove commented code.

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 3]
```diff
@@ -808,7 +812,7 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
                 if (isElectron) {
                     this.router.navigate(['/login']);
                 } else {
-                    window.location.href = (environment.production) ? `https://stage.giddh.com/login/?action=logout` : `https://test.giddh.com/login/?action=logout`;
+                    window.location.href = (environment.production) ? `https://beta.giddh.com/login/?action=logout` : `https://test.giddh.com/login/?action=logout`;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:815]: Please revert the url

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

**Review by shalinee-walkover — APPROVED** [review]
Approve

**Review by rohit-walkover — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**